system-tests/_support/index.js

```diff
@@ -159,6 +159,30 @@ Cypress.addChildCommand("contents", function(elements) {
     .get();
 });
 
+/* Sets given JSONString as Value for ACE Editor
```

Comment:
@tnguyen0 since you are using VS code you can use https://marketplace.visualstudio.com/items?itemName=joelday.docthis